### PR TITLE
fix(php): composer dependency

### PR DESCRIPTION
**Pull request by damcou**
### Changes included:

Added a small change on the dependencies of the composer.json since there's an issue when I try to install the PHP client on a raw laravel app.

## 🧪 Test

CI

### CODE REVIEW

File: templates/php/composer.mustache [hunk 1]
```diff
@@ -34,7 +34,7 @@
         "psr/simple-cache": "^1.0 || ^2.0 || ^3.0"
     },
     "require-dev": {
-        "friendsofphp/php-cs-fixer": "^3.5.0",
+        "friendsofphp/php-cs-fixer": "^2.0 || ^3.5.0",
```
Comment (shortcuts) [inline templates/php/composer.mustache:37]: Shouldn't we require at least 3.5.0? I believe the syntax can change a lot from a major to an other

Comment (damcou) [reply templates/php/composer.mustache:37]: I tested locally, and the v3.x is installed (and no changes are generated in the files), I just want to see if I can install the new php client in a laravel app (currently it's possible with the current php client with this dependency: https://github.com/algolia/algoliasearch-client-php/blob/master/composer.json#L23)

Comment (shortcuts) [reply templates/php/composer.mustache:37]: Ah ok if it was like this it make sense to keep it indeed

**Review by shortcuts — APPROVED** [review]
nice

**Pull request merged**